_data/sidebars/es_lb4_sidebar.yml

```diff
@@ -0,0 +1,424 @@
+## definition of `menuitem`===========
+# title: string
+# url: string
+# output: [ enum{web, pdf} ] (expressed as comma separated string in YAML)
+# children?: [ menuitem ]
+
+## top level property=================
+# entries: [ menuitem ]
+
+title: LoopBack 4
+url: index.html
+children:
```

Comment:
since this page is translated, please add `translated: true` before `children:` and add `translated: false` for the other entries